arcgis-ios-sdk-samples/Maps/Show location history/README.md

```diff
@@ -0,0 +1,43 @@
+# Show location history
+
+Display your location history on the map.
+
+![](image1.png)
+
+## Use case
+
+You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. An unmapped trail, for example, could be added to the map using this technique.
+
+## How to use the sample
+
+Tap 'Start tracking' to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Tap 'Stop tracking' to stop updating the location history.
+
+## How it works
+
+1. If necessary, request location permission from the operating system.
+2. Create a graphics overlay to show each point and another graphics overlay for displaying the route line.
+3. Create a `AGSCLLocationDataSource` and start it to begin receiving location updates.
```

Comment:
```suggestion
3. Create an `AGSCLLocationDataSource` and start it to begin receiving location updates.
```

Reply:
I revisited this entire line in [`864bcca3`](https://github.com/Esri/arcgis-runtime-samples-ios/pull/751/commits/864bcca3c9f54e69c6b205c438e852502e7eae32).